rust/pattern/variable/builder/type_.rs

```diff
@@ -32,6 +33,7 @@ pub trait TypeConstrainable {
 }
 
 pub trait TypeVariableBuilder: Sized {
+    fn abstract_(self) -> TypeVariable;
```

Comment:
`is_abstract()` to me implies a getter, so dropping the `is` here.

Reply:
According to the tests, the sample usage of this is
```rs
typeql_define!(
        type_("concrete-type").sub("entity"),
        type_("abstract-type").sub("entity").abstract_()
    );
```
which does, indeed, read better than `is_abstract`.

How's it named in Java? If we're using `isAbstract` for the same method in Java, we should update that too, I think. Although, that would be a breaking API change for existing users. But it seems like a correct decision.

Reply:
The changes in the Java version would propagate to e.g. TypeDB core, so I'm not touching it yet. I think that's way out of scope.

Reply:
In that case, let's just raise an issue for now.

Reply:
https://github.com/vaticle/typeql/issues/241